dist/tools/pic32prog/doc.md

```diff
@@ -0,0 +1,136 @@
+Flashing on Linux using pickit-3
+================================
+
+Allow flashing a pic32-wifire from linux using https://github.com/sergev/pic32prog
+
+
+Informations come from this comment
+
+https://github.com/RIOT-OS/RIOT/pull/6092#issuecomment-261987955
+
+
+Steps
+-----
+
+### Setup your computer
+
+I ran this on Ubuntu 16.04
+
+* Install Virtualbox
+* Add yourself to the 'vboxusers' group
+  `sudo usermod -a -G vboxusers your_username`
+* Reboot (was necessary for the ext-pack may be possible without)
+
+### Setup Windows VM
+
+You can download a free 90 days valid windows virtualbox image from:
+
+    https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/
+
+I downloaded a Windows 10 virtualbox image
+
+* Create a machine in virtualbox in File-Import Appliance.
+  * optional: disabled network
```

Comment:
I will remove the parts about networking and shared folder as these are just because it is not required to connect your windows to the internet and that I tend to not connect them without antivirus but it should not be the problem of the tutorial.